css/styles.css

```diff
@@ -97,3 +102,219 @@ header {
 .m-icon::before {
   content: url("icon-medium.png");
 }
+
+.works {
+  width: 375px;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-columns: auto;
+  padding: 120px 24px 96px;
+  background-color: white;
+}
+
+.works > h1 {
+  color: #1c1a19;
+  font-family: "Crete Round", sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+  line-height: 52px;
+  border-bottom: 1px solid #212121;
+  padding-bottom: 32px;
+}
+
+.featured-work {
+  margin-top: 40px;
+  margin-bottom: 24px;
+  display: flex;
+  flex-direction: column;
+}
+
+.featured-work h2 {
+  color: #091e42;
+  font-family: "Crete Round", sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+  margin: 18px 0 0 0;
+}
+
+.featured-work p {
+  color: #091e42;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  margin-top: 8px;
+}
+
+.tags {
+  display: flex;
+  list-style: none;
+  margin: 10px 0;
+  padding: 0;
+  gap: 12px;
+}
+
+.tags-elements {
+  text-decoration: none;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  color: #091e42;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+.tags-elements:hover {
+  opacity: 0.5;
+}
+
+.button {
+  background-color: #ff6b00;
+  padding: 12px;
+  align-self: flex-start;
+  margin: 12px 0 0 0;
+  color: #ffff;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.button:hover {
+  opacity: 0.5;
+}
```

Comment:
- [ ] Please consider following the correct button styling: Active, Hover, and Pressed as seen in your chosen Figma template. You can get their correct color and design from Figma. Below is a screenshot of them:

<img width="265" alt="Loom_sqhPK3wjI8" src="https://user-images.githubusercontent.com/81468675/148486717-a7072938-24da-449f-a328-74c3de52bb69.png">